core/bin/external_node/src/config/mod.rs

```diff
@@ -136,6 +136,14 @@ pub struct OptionalENConfig {
     latest_values_cache_size_mb: usize,
     /// Enabled JSON RPC API namespaces.
     api_namespaces: Option<Vec<Namespace>>,
+    /// Whether to support methods installing filters and querying filter changes.
+    ///
+    /// Rationale for setting this to `false`:
+    /// Filters are local to the specific node they were created at. Meaning if
+    /// there are multiple nodes behind a load balancer the client cannot reliably
+    /// query the previously created filter as the request might get routed to a
+    /// different node.
+    pub filters_enabled: Option<bool>,
```

Comment:
- Why is this value optional? It's possible to set a `bool` value to the default using `#[serde(default = ..)]`, like for other fields in this struct. IMO, it'd be simpler to reason about and more idiomatic. Ditto for a similar field in other configs.
- Bikeshedding: IMO, it would be easier to reason about an inverse variable (`filters_disabled`). In this case, the absence of the variable is equivalent to a `false` value, which looks more intuitive. Additionally, you'd be able to use `#[serde(default)]` instead of `#[serde(default = ..)]` 🙃

Reply:
Fixed in https://github.com/matter-labs/zksync-era/pull/1078/commits/5df4df4ec48805e91c24e23945087d364c0b8c47 and https://github.com/matter-labs/zksync-era/pull/1078/commits/53c747832abc619e453f94a59c703132f512b973

> IMO, it would be easier to reason about an inverse variable (filters_disabled)

I usually try to avoid "negative" config values as you can sometimes find yourself using it with negation and it is hard to reason about double negation. Not a strong opinion on this though